pkg/rtc/wrappedreceiver.go

```diff
@@ -103,6 +103,14 @@ func (r *WrappedReceiver) Codecs() []webrtc.RTPCodecParameters {
 	return codecs
 }
 
+func (r *WrappedReceiver) DeleteDownTrack(participantID livekit.ParticipantID) {
```

Comment:
Adding this one function of `sfu.TrackReceiver` interface in the wrapper as this can get called even when not bound now (with the change in downtrack.go). And a successful bind is when `DetermineCodec` function ☝️ is called and a receiver set up. And the down track added to the determined receiver.